src/Tempest/Cache/src/PHPCacheInitializer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Cache;
+
+use Symfony\Component\Cache\Adapter\PhpFilesAdapter;
+use Tempest\Container\Container;
+use Tempest\Container\Initializer;
+use Tempest\Container\Singleton;
+
+final readonly class PHPCacheInitializer implements Initializer
+{
+    #[Singleton(tag: 'php')]
```

Comment:
Nope, bad idea, changed my mind. Gonna go with the interface + trait combo so that I can implement my own `ViewCache` and `DiscoveryCache` where needed.